example-apigateway/Setup.hs

```diff
@@ -0,0 +1,2 @@
+import Distribution.Simple
```

Comment:
You can deploy more than one function from a single Haskell project.

Reply:
I'm not sure I understand what change you want me to make.

Reply:
You can put both example functions (the old one and the new API Gateway one) into the single `example-project` as two different (Stack) components.